### DISCUSSION TITLE
Determine interactions of multiple sgRNAs on expression?

### DISCUSSION THREAD

**Discussion by josiegleeson** [Katsevich-Lab/sceptre#134] [Q&A] [answered]
Hi, I'm relatively new to the field so apologies if this is a flawed question. I was wondering if its possible in a high MOI experiment using sgRNAs to attenuate gene expression, to study the potential synergistic or antagonistic effect of sgRNAs on downstream expression? Say for example I want to study whether two genes interact with each other so I'm targeting their expression using CRISPRa/i with multiple sgRNAs and observing whether downstream genes are differentially expressed, is it possible to assess the effect of sgRNAs individually, and in combination with each other?

Thanks for any help or advice!

**Comment by ekatsevi** [comment, marked as answer]
Unfortunately, sceptre does not currently support testing for interactions. You might want to try out [GLiMMIRS](https://github.com/mcvickerlab/GLiMMIRS), which was designed for this purpose.

**Reply by josiegleeson** [reply]
Oh perfect, thanks for the recommendation!
